src/controller/model/endpoint.ts

```diff
@@ -573,6 +573,33 @@ class Endpoint extends Entity {
         }
     }
 
+    public async writeStruct(clusterKey: number | string, payload: KeyValue, options?: Options): Promise<void> {
```

Comment:
Please rename this to `writeStructured` (as this is how it is called in the ZCL)

Reply:
Updated!